packages/website/ts/utils/utils.ts

```diff
@@ -309,6 +309,17 @@ export const utils = {
             containerId,
         });
     },
+    web3ProviderToString(provider: Web3Provider): string {
+        let parsedProviderName = provider.constructor.name;
+        if (provider.constructor.name === 'MetamaskInpageProvider') {
+            parsedProviderName = 'METAMASK';
+        } else if (provider.constructor.name === 'EthereumProvider') {
+            parsedProviderName = 'MIST';
+        } else if ((provider as any).isParity) {
+            parsedProviderName = 'PARITY';
+        }
```

Comment:
Maybe add a default case of `UNKNOWN` to imply there is a web3 but we don't know what it is (yet)

Reply:
I originally had that, but took it out and replaced it with just the constructor name. I feel like gives us more info and still implies that "UNKNOWN" case

Reply:
Yeah I'm just worried about anyone else who is minified, might end up with a bunch of `o`, `a` etc.

Reply:
I feel you. I think we can just map those to "UNKNOWN" offline and we might also capture some understandable constructor names (e.g. MetamaskInpageProvider). I'm down to change this though if ppl think it's better to have 'UNKNOWN' be mapped explicitly.